core/src/main/java/com/orange/ouds/core/component/OudsNavigationBar.kt

```diff
@@ -0,0 +1,511 @@
+ * content that will have the height of OudsNavigationBar. For this, please use `OudsNavigationBarHeight` constant.
+ *
+ * @param modifier [Modifier] applied to the navigation bar.
+ * @param translucent Whether the navigation bar should be translucent.
+ * @param windowInsets Window insets of the navigation bar.
+ * @param content Content of the navigation bar.
+ *
+ * @sample com.orange.ouds.core.component.samples.OudsNavigationBarSample
+ */
+@Composable
+fun OudsNavigationBar(
+    modifier: Modifier = Modifier,
+    translucent: Boolean = false,
+    windowInsets: WindowInsets = NavigationBarDefaults.windowInsets,
+    content: @Composable RowScope.() -> Unit
+) {
+    val topBorderColor = OudsTheme.colorScheme.border.minimal
+    with(OudsTheme.componentsTokens.bar) {
+        NavigationBar(
+            modifier = modifier
+                .focusProperties { canFocus = false }
+                .drawBehind {
+                    drawLine(
+                        color = topBorderColor,
+                        start = Offset(x = 0f, y = 0f),
+                        end = Offset(x = size.width, y = 0f),
+                        strokeWidth = 1.dp.toPx()
+                    )
+                },
+            containerColor = if (translucent && Build.VERSION.SDK_INT > Build.VERSION_CODES.S_V2) colorBgTranslucent.value else colorBgOpaque.value,
+            contentColor = colorContentUnselectedEnabled.value,
+            windowInsets = windowInsets,
+            content = content
+        )
+    }
+}
+
+/**
+ * An OUDS navigation bar item.
+ *
+ * It must be used in [OudsNavigationBar].
+ *
+ * The recommended configuration for an [OudsNavigationBarItem] depends on how many items there are inside a [OudsNavigationBarItem]:
+ * - Three destinations: Display icons and text labels for all destinations.
+ * - Four destinations: Active destinations display an icon and text label. Inactive destinations display icons, and text labels are recommended.
+ * - Five destinations: Active destinations display an icon and text label. Inactive destinations use icons, and use text labels if space permits.
+ *
+ * A [OudsNavigationBarItem] always shows text labels (if it exists) when selected. Showing text labels if not selected is controlled by [alwaysShowLabel].
+ *
+ * @param selected Whether this item is selected or not.
+ * @param onClick Called when this item is clicked.
+ * @param icon Icon of the item.
+ * @param modifier [Modifier] applied to the navigation bar item.
+ * @param label Label of the item.
+ * @param badge Optional badge display on the item icon.
+ * @param alwaysShowLabel Whether the label should always be shown.
+ * @param interactionSource [MutableInteractionSource] that will be used to dispatch events when this item is pressed, hovered or focused.
+ *
+ * @sample com.orange.ouds.core.component.samples.OudsNavigationBarSample
+ */
+@Composable
+fun RowScope.OudsNavigationBarItem(
```

Comment:
Should we keep this composable method or should we use an `OudsNavigationBarItem` class instead, with an `items` parameter in `OudsNavigationBar`? Is the design OK with the fact that developers will be able to change colors and paddings of items?